data/json/items/tool/explosives.json

```diff
@@ -1246,6 +1283,7 @@
     "weight": "710 g",
     "//": "Only one third (410 g) of a whole pipe is being used",
     "volume": "500 ml",
+    "longest_side": "18 cm",
```

Comment:
Where is this length from? (It'd be nice if you added a length for the pipe - see items/resources/metal.json.)

Reply:
Took a look at some photos of pipe bombs, looked to be about that size.  Still going to keep adding some more longest_sides, just need to carve out some time.